resources/ts/Components/Card.tsx

```diff
@@ -87,36 +87,38 @@ export const Card = ({
       <div
         className={classNames(
           "flex flex-col",
-          textCenter && "text-center",
-          !border && "gap-5 items-center"
+          textCenter && "text-center items-center",
+          !border && "gap-5"
         )}
       >
-        <div
-          className={classNames(
-            "flex justify-between",
-            border &&
-              `${!headless ? "border-b border-publiq-gray-300 max-sm:px-2 p-5" : "py-3"}`
-          )}
-        >
+        {(title || icon || badge || iconButton) && (
           <div
-            className={`flex items-center gap-3 ${headless ? "h-16" : "h-11"}`}
-          >
-            {icon}
-            {clickableHeading ? (
-              <Link href={`${translateRoute("/integrations")}/${id}`}>
-                {heading}
-              </Link>
-            ) : (
-              heading
-            )}
-            {!!badge && (
-              <span className=" text-publiq-gray-medium-dark bg-publiq-gray-light uppercase border border-publiq-gray-medium-dark text-xs font-medium  mr-2 px-2.5 py-0.5 rounded">
-                {badge.replaceAll("-", " ")}
-              </span>
+            className={classNames(
+              "flex justify-between",
+              border &&
+                `${!headless ? "border-b border-publiq-gray-300 max-sm:px-2 p-5" : "py-3"}`
             )}
+          >
+            <div
+              className={`flex items-center gap-3 ${headless ? "h-16" : "h-11"}`}
+            >
+              {icon}
+              {clickableHeading ? (
+                <Link href={`${translateRoute("/integrations")}/${id}`}>
+                  {heading}
+                </Link>
+              ) : (
+                heading
+              )}
+              {!!badge && (
```

Comment:
The Card component is getting a bit messy.

Moving certain parts out onto variables and using early returns if possible could cleanup the code a bit